### PR TITLE
test(cypress): add authentication tests

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Add Cypress tests for logging in and authentication from legacy.

## QA

No QA, CI should pass.

## Fixes

Fixes: canonical-web-and-design/maas-ui#992.
Fixes: canonical-web-and-design/maas-ui#429.

### CODE REVIEW

File: integration/cypress/integration/utils.ts [hunk 1]
```diff
@@ -22,7 +21,3 @@ export const generateMac = () =>
   );
 
 export const generateEmail = () => `${nanoid()}@example.com`;
-
-export const makeUIURL = (path: string) => {
```
Comment (huwshimi) [inline integration/cypress/integration/utils.ts:26]: Drive-by replacement with the shared util.

File: yarn.lock [hunk 2]
```diff
@@ -9972,7 +9972,7 @@ jest-environment-node@^26.5.0:
     jest-mock "^26.5.0"
     jest-util "^26.5.0"
 
-jest-fetch-mock@3.0.3, jest-fetch-mock@^3.0.3:
+jest-fetch-mock@3.0.3:
```
Comment (huwshimi) [inline yarn.lock:9975]: Drive-by update of the pinned version.

**Review by Caleb-Ellis — APPROVED** [review]
LGTM!
